### PR TITLE
e2ee: show warning if not all participants support E2EE

### PR DESCRIPTION
**Pull request by saghul**
Refs: https://github.com/jitsi/lib-jitsi-meet/pull/1108

### CODE REVIEW

File: lang/main.json [hunk 1]
```diff
@@ -178,6 +178,7 @@
         "e2eeDescription": "<p>End-to-End Encryption is currently <strong>EXPERIMENTAL</strong>. Please see <a href='https://jitsi.org/blog/e2ee/' target='_blank'>this post</a> for details.</p><br/><p>Please keep in mind that turning on end-to-end encryption will effectively disable server-side provided services such as: recording, live streaming and phone participation. Also keep in mind that the meeting will only work for people joining from browsers with support for insertable streams.</p>",
         "e2eeLabel": "Key",
         "e2eeTitle": "End-to-End Encryption",
+        "e2eeWarning": "<br /><p><strong>WARNING:</strong> Not all participants in this meeeting seem to have support for End-to-En encryption. If you enable it they won't be able to see nor hear you.</p>",
```
Comment (JonathanLennox) [inline lang/main.json:181]: Spelling: "meeeting", "End-to-En"

Comment (saghul) [reply lang/main.json]: F*ckinfg macbook keyboard. Fixed!

File: lang/main.json [hunk 2]
```diff
@@ -178,6 +178,7 @@
         "e2eeDescription": "<p>End-to-End Encryption is currently <strong>EXPERIMENTAL</strong>. Please see <a href='https://jitsi.org/blog/e2ee/' target='_blank'>this post</a> for details.</p><br/><p>Please keep in mind that turning on end-to-end encryption will effectively disable server-side provided services such as: recording, live streaming and phone participation. Also keep in mind that the meeting will only work for people joining from browsers with support for insertable streams.</p>",
         "e2eeLabel": "Key",
         "e2eeTitle": "End-to-End Encryption",
+        "e2eeWarning": "<br /><p><strong>WARNING:</strong> Not all participants in this meeting seem to have support for End-to-End encryption. If you enable it they won't be able to see nor hear you.</p>",
```
Comment (fippo) [inline lang/main.json:181]: nit: see or hear?

Comment (saghul) [reply lang/main.json:181]: I checked: " used to introduce a further negative statement: " hence I went for it.

**Pull request merged**